sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/src/ObjectAnchorsConversionClientOptions.cs

```diff
@@ -16,6 +18,11 @@ public class ObjectAnchorsConversionClientOptions : ClientOptions
     {
         internal string Version { get; }
 
+        /// <summary>
+        /// Gets the list of supported asset file types
+        /// </summary>
+        internal IEnumerable<AssetFileType> SupportedAssetFileTypes { get; }
```

Comment:
This can probably just be an array or even a HashSet to make it easier to look up.